packages/network_info_plus/network_info_plus/example/lib/main.dart

```diff
@@ -86,49 +86,17 @@ class _MyHomePageState extends State<MyHomePage> {
   }
 
   Future<void> _initNetworkInfo() async {
-    String? wifiName,
-        wifiBSSID,
-        wifiIPv4,
-        wifiIPv6,
-        wifiGatewayIP,
-        wifiBroadcast,
-        wifiSubmask;
+    String? wifiName, wifiBSSID, wifiIPv4, wifiIPv6, wifiGatewayIP, wifiBroadcast, wifiSubmask;
 
     try {
-      if (!kIsWeb && Platform.isIOS) {
```

Comment:
@jpnurmi what was the reason for implementing the location service request, before getting the wifi's name and bssid?
The methods `getLocationServiceAuthorization` and `requestLocationServiceAuthorization` are deprecated in our plugin, I replaced them with `permission_handler`'s methods, but eventually didn't understand the necessity for them.

Reply:
hmm, this doesn't ring any bells. `network_info_plus` was split off from `connectivity_plus`, which was inherited from `flutter/plugins/connectivity`. i suspect the logic originates from somewhere there...